djangochannelsrestframework/observer/model_observer.py

```diff
@@ -182,6 +182,10 @@ def serialize(self, instance, action, **kwargs) -> Dict[str, Any]:
         else:
             message_body["pk"] = instance.pk
 
+        # Check if the pk of the model is UUID type and convert it to a string
+        if isinstance(message_body['pk'], UUID):
+            message_body.update({'pk': str(message_body['pk'])})
```

Comment:
Might be better to use `json.dumps` (I believe this will have the required result for UUID and will work in the case of other data types as well?)

Reply:
also the result of self._serliser might not include a PK field. 

I think it would be better to move this login up into the `else` bock 

```
else:
    message_body["pk"] = json.dumps(instance.pk)
```